src/server.py

```diff
@@ -108,6 +108,7 @@ def play_sound(self, sound_event_id, connection, sender, path, iface,
             invocation.return_dbus_error(
                 self._DBUS_UNKNOWN_SOUND_EVENT_ID,
                 "sound event with id %s does not exist" % sound_event_id)
+            self.release_if_no_players()
```

Comment:
Can you give some higher level descriptions in the commit description about what this does, and what we fix? What happened before the fix?

Reply:
I realized this is covered in #8 (with a timeout). I will remove this commit.